packages/client/src/components/error-stub/error-stub.tsx

```diff
@@ -0,0 +1,23 @@
+import { FC } from 'react';
+
+import style from './error-stub.module.pcss';
+
+type ErrorStubProps = {
+  title: string;
+  message: string;
+  children?: JSX.Element | JSX.Element[];
+};
+
+const ErrorStub: FC<ErrorStubProps> = (props) => {
```

Comment:
```suggestion
const ErrorStub: FC<ErrorStubProps> = ({ title, message, children }) => {
```
можно деструктуризацию сразу в аргументе сделать